src/main/java/nextstep/subway/domain/From10To50Policy.java

```diff
@@ -0,0 +1,29 @@
+package nextstep.subway.domain;
+
+public class From10To50Policy implements SubwayFarePolicy {
+    private static final int EXTRA_FARE = 100;
+    private static final int EXTRA_STEP = 5;
+    private static final int MIN_DISTANCE = 10;
+    private static final int MAX_DISTANCE = 50;
+
+    @Override
+    public int apply(int currentFare, int distance) {
```

Comment:
```suggestion
    public int apply(int currentFare, int distance) {
        if (distance <= MIN_DISTANCE) {
            return currentFare;
        }

        if (distance <= MAX_DISTANCE) {
            return currentFare + calculateOverFare(distance - MIN_DISTANCE);
        }

        return currentFare + calculateOverFare(MAX_DISTANCE - MIN_DISTANCE);
    }
```

예외가 발생 할 수 없겠네요.
이렇게 쓰면 더 간결하지 않을까요?